Source/Core/DolphinQt/Achievements/AchievementSettingsWidget.cpp

```diff
@@ -75,6 +81,20 @@ void AchievementSettingsWidget::CreateLayout()
          "achievements.<br><br>Unofficial achievements may be optional or unfinished achievements "
          "that have not been deemed official by RetroAchievements and may be useful for testing or "
          "simply for fun."));
+  m_common_hardcore_enabled_input = new ToolTipCheckBox(tr("Enable Hardcore Mode"));
+  m_common_hardcore_enabled_input->SetDescription(
+      tr("Enable Hardcore Mode on RetroAchievements.<br><br>Hardcore Mode is intended to provide "
+         "an experience as close to gaming on the original hardware as possible. RetroAchievements "
+         "rankings are primarily oriented towards Hardcore points (Softcore points are tracked but "
+         "not as heavily emphasized) and leaderboards require Hardcore Mode to be on.<br><br>To "
+         "ensure this experience, the following features will be disabled, as they give emulator "
+         "players an advantage over console players:<br>- Loading states<br>-- Saving states is "
+         "allowed<br>- Emulator speeds below 100%<br>-- Includes frame step<br>-- Turbo is "
```

Comment:
In the UI we call it frame advance, not frame step.

Also, "includes" is a little ambiguous. Does that mean it's allowed, or that it's included in "emulator speeds below 100%"?